Example/Backpack/Views/CollectionView Cells/IconsPreviewCollectionViewCell.swift

```diff
@@ -52,7 +52,16 @@ class IconsPreviewCollectionViewCell: UICollectionViewCell {
     }
 
     // MARK: private
-    private func setup() {
+    private func setupIconView(size: BPKIconSize) {
+        contentView.subviews.forEach({ $0.removeFromSuperview() })
+
+        switch size {
+        case .small:
+            imageView = BPKSmallIconView(iconName: BPKSmallIconName(rawValue: icon?.rawValue ?? "bar"))
```

Comment:
Why "bar"?

Reply:
Can we `fatalError` instead?

Reply:
This is called before `icon` is set though

Reply:
Of course 🙈